rollup/dist.config.js

```diff
@@ -13,9 +12,10 @@ export default [
         input: ['./src/index.ts', './src/internal.ts'],
         output: [
             {
-                dir: `${OUTPUT_DIR}`,
-                format: 'cjs',
-                sourcemap: true
+                dir: OUTPUT_DIR,
+                entryFileNames: '[name].cjs',
+                chunkFileNames: '[name]-[hash].cjs',
```

Comment:
Why chunks? Are you splitting it?

Reply:
This is internally used by rollup. By default it will generate them with plain .js extension, but if we switch to module, we need to give them explicit .cjs extension (as they are commonjs).

Example:
```
❯ la dist
Permissions Size User       Group Date Modified Git Name
.rw-r--r--   585 mschuwalow staff  6 Mar 17:09   -I index.cjs
.rw-r--r--   276 mschuwalow staff  6 Mar 17:09   -I index.cjs.map
.rw-r--r--   307 mschuwalow staff  6 Mar 17:09   -I index.cjs.mjs
.rw-r--r--  2.9k mschuwalow staff  6 Mar 17:09   -I index.d.ts
.rw-r--r--   66k mschuwalow staff  6 Mar 17:09   -I initializer-me88Mdta.cjs
.rw-r--r--   64k mschuwalow staff  6 Mar 17:09   -I initializer-me88Mdta.cjs.map
.rw-r--r--   584 mschuwalow staff  6 Mar 17:09   -I internal.cjs
.rw-r--r--   107 mschuwalow staff  6 Mar 17:09   -I internal.cjs.map
.rw-r--r--   470 mschuwalow staff  6 Mar 17:09   -I internal.cjs.mjs
.rw-r--r--  1.1k mschuwalow staff  6 Mar 17:09   -I internal.d.ts
.rw-r--r--  6.6k mschuwalow staff  6 Mar 17:09   -I standard-live-connect.d-CGhFVSFd.d.ts
```